src/System.Text.Encodings.Web/ref/System.Text.Encodings.Web.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
+  <PropertyGroup>
+    <ProjectGuid>{41648C6D-D431-478D-8228-7EB68714541C}</ProjectGuid>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release;uap-Debug;uap-Release</Configurations>
```

Comment:
Why do you need the non-netstandard configurations? Seems to me that the netstandard one should suffice as you are not really if-deffing out anything but the references.